exercise-03/exercise-03.js

```diff
@@ -10,7 +10,9 @@
  * ```
  */
 const distinct = (iterable) => {
```

Comment:
I believe this file shouldn't be a part of this PR but a PR dealing with `exercise-03`